.github/workflows/ci.yml

```diff
@@ -28,9 +28,72 @@ jobs:
           cd templates/project-ts
           npm ci || true
           npm run test
-          rm -rf node_modules
         env:
           CI: true
+  sudoku-example:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
+    steps:
+      - name: Set up NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Git checkout
+        uses: actions/checkout@v2
+      - name: NPM ci, build, & test
+        run: |
+          npm ci
+          npm run build --if-present
```

Comment:
I'm not sure why this was in the CI script, but I copied it here. I guess to anticipate the possibility of an `NPM `build` script being added?